components/shared/AIToolContent.tsx

```
@@ -0,0 +1,172 @@
+                      <div className="h-2 w-2 rounded-full bg-indigo-500 mt-2 mr-2 flex-shrink-0"></div>
+                      <p className="mt-0">{content}</p>
+                    </div>
+                  );
+                }
+                
+                // Regular paragraphs with subtle highlights for important phrases
+                if (line.trim()) {
+                  // Highlight text in backticks
+                  const formattedText = line.replace(/`(.*?)`/g, '<span class="px-1 bg-indigo-50 text-indigo-700 rounded font-mono text-sm">$1</span>');
+                  
+                  return <p key={index} className="my-1.5" dangerouslySetInnerHTML={{ __html: formattedText }}></p>;
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)